bigbluebutton-html5/imports/api/presentations/server/handlers/presentationConversionDone.js

```diff
@@ -0,0 +1,29 @@
+import { check } from 'meteor/check';
+import Presentations from '/imports/api/presentations';
+
+export default function handlePresentationConversionDone({ payload }) {
+  const meetingId = payload.meeting_id;
+  const presentationId = payload.presentation.id;
+  const status = payload.message_key;
+
+  check(meetingId, String);
+  check(presentationId, String);
+
```

Comment:
Im pretty sure theres sometimes that the status is not sended. But I will double-check it.